spec/mailers/consolidated_mailer_spec.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe ConsolidatedMailer, type: :mailer do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Remove this or?

Reply:
I'm planning to write some proper specs next week.